homework/Truong_Duong/Lesson_13/Task2.js

```diff
@@ -0,0 +1,23 @@
+function solution(S) {
+  let res = 0;
+
+  while (S[0] === '0') {
+    S = S.slice(1);
+  }
+
+  while (S !== '0') {
+    const size = S.length;
+    if (S[size - 1] === '0') {
+      S = S.slice(0, size - 1);
+    } else {
+      S = S.slice(0, size - 1) + '0';
+    }
+    res += 1;
+  }
+
+  return res;
+}
+
+console.log(solution('011100') === 7);
```

Comment:
is there a better and simpler way? hint: 
- /2 if event <=> remove last 0, res++
- -1 if odd <=> remove last 0, res+=2

=> the problem becomes counting #0 and #1.